hatch/src/generators/tup/tests/platform.rs

```diff
@@ -1,19 +1,34 @@
-use generators::tup::platform::linux::to_string as linux_to_string;
-use generators::tup::platform::mac_os::to_string as mac_os_to_string;
-use generators::tup::platform::windows::to_string as windows_to_string;
+use generators::tup::platform::linux;
+use generators::tup::platform::mac_os;
+use generators::tup::platform::windows;
 
 #[test]
-fn build_linux() {
-    assert_eq!("STATIC = a\nSHARED = so", linux_to_string());
+fn linux_file_name() {
+    assert_eq!("linux.tup", linux::file_name())
 }
 
 #[test]
-fn build_macos() {
-    assert_eq!("STATIC = a\nSHARED = so", mac_os_to_string());
+fn generate_linux() {
+    assert_eq!("STATIC = a\nSHARED = so", linux::make_string());
 }
 
 #[test]
-fn build_windows() {
+fn mac_os_file_name() {
+    assert_eq!("macosx.tup", mac_os::file_name())
+}
+
+#[test]
+fn generate_mac_os() {
+    assert_eq!("STATIC = a\nSHARED = so", mac_os::make_string());
+}
+
+#[test]
+fn windows_file_name() {
+    assert_eq!("win32.tup", windows::file_name())
+}
+
+#[test]
+fn generate_windows() {
```

Comment:
are these tests valuable? We are comparing hardcoded strings against hardcoded strings.